content/source/docs/glossary.html.md

```diff
@@ -871,9 +895,11 @@ Available Terraform versions are configured at a per-instance level in [Private
 
 Also "input variables".
 
-Key/value pairs used in a [run][]. Terraform [modules][] can declare variables to allow customization. For child modules, the parent module provides a value when calling the module; for the [root module][], values are provided at run time.
+In general-purpose programming, a variable is a symbolic name associated with a value.
+
+In Terraform, "variables" almost always refers to _input variables,_ which are key/value pairs used in a [run][]. Terraform [modules][] can declare variables to allow customization. For child modules, the parent module provides a value when calling the module; for the [root module][], values are provided at run time.
 
-TFE lets you set root input variables in a [workspace][], so all collaborators can use the same values. Variable values marked as "sensitive" become unreadable in the UI and API, and all variable values are protected by Vault.
+TFE lets you set values for root input variables in a [workspace][], so all collaborators can use the same values. Variable values marked as "sensitive" become unreadable in the UI and API, and all variable values are protected by Vault.
```

Comment:
I wonder if it's worth linking out to the other docs on this?

Reply:
There's a link to the TFE workspace variables docs right below!